docs/pages/modules/mocking.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Mocking native calls in Expo modules'
+sidebar_title: Mocking
+description: Learn about mocking native calls in Expo modules.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Grid01Icon } from '@expo/styleguide-icons';
+import { Terminal } from '~/ui/components/Snippet';
+
+The recommended way to write unit tests for an Expo project is to use [Jest](https://jestjs.io/) and the `jest-expo` preset.
+
+<BoxLink
+  title="Unit testing"
+  description="Learn how to set up and configure the jest-expo package to write unit tests and snapshot tests for a project."
+  href="/develop/unit-testing/"
+  Icon={Grid01Icon}
+/>
```

Comment:
Let's move this below tip and add a new section for it. As a reader, I'm a bit confused by seeing this as the first thing on the page and after the boxlink I see context on writing unit test for native code.

Reply:
That works, I was thinking something like: here's how to mock in general, and here's a specific case, but your suggestion also makes sense.

Reply:
Thanks! 

May be we can add a generic section like "More" and in the description of the box link mention that for a more general purpose mocking, see our unit testing guide.